src/toolkits/drawing_classifier/dc_data_iterator.cpp

```diff
@@ -0,0 +1,177 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include <model_server/extensions/additional_sframe_utilities.hpp>
+#include <toolkits/drawing_classifier/dc_data_iterator.hpp>
```

Comment:
Please include `dc_data_iterator.hpp` first, in its own grouping

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done.

A quick follow-up: Should `dc_data_iterator.hpp` be included as "dc_data_iterator.hpp" or as <toolkits/drawing_classifier/dc_data_iterator.hpp> ? I got conflicting information on this.

Reply:
The latter, since we're under `src`. Things are a bit weirder under `test`. We should use fully qualified paths when possible